### PR TITLE
Fyle 2khjeya-fix for overlapping pills

### PR DESCRIPTION
**Pull request by SowmyaFyle**
image

image

image

### CODE REVIEW

File: src/app/shared/components/expenses-card/expenses-card.component.scss [hunk 1]
```diff
@@ -265,11 +265,13 @@
     margin-top: 6px;
     min-width: 117px;
     white-space: nowrap;
+    position: absolute;
```
Comment (yksolanki9) [inline src/app/shared/components/expenses-card/expenses-card.component.scss:268]: Can we not use absolute positioning here?

Comment (Julias0) [reply src/app/shared/components/expenses-card/expenses-card.component.scss:268]: +1

**Review by Julias0 — CHANGES_REQUESTED** [review]
Comments added

File: src/app/shared/components/expenses-card/expenses-card.component.scss (hunk 1 above)
Comment (SowmyaFyle) [reply src/app/shared/components/expenses-card/expenses-card.component.scss]: removed

**Review by Julias0 — APPROVED** [review]

**Pull request merged**